docker-compose.yaml

```diff
@@ -7,7 +7,7 @@ version: '3.7'
 services:
   gooddata-cn-ce:
     # Use stable releases represented by tags like 1.5
-    image: gooddata/gooddata-cn-ce:1.7.2
+    image: gooddata/gooddata-cn-ce:dev_latest
```

Comment:
This does not look consistent with comment one line above.

Reply:
True, thanks. I will fix it 👍